mapping/sustainability.yaml

```diff
@@ -742,49 +759,24 @@
         Eg:\
         Domestic mobilisations \[value or NA\]
         Private Finance \[value or NA\]
-    mapping: |-
-        Project level:
-
-        For each co-financing arrangement add a new `finance` object to the `budget.finance` array. Map the value and currency to the objects `.value`. Add the financing party to the parties array and add `funder` to their `roles` array. Map the funders `id` and `name` to the `.financingParty` object in the `finance` object.
+    mapping: The OC4IDS data model provides the information needed to calculate co-finance ratios by modelling a project's individual financing arrangements. For more information, see the mappings for [amount of investment](climate-finance-amount-of-investment), [funding source](climate-finance-funding-source), [Green Climate Fund Accredited Entity](climate-finance-green-climate-fund-accredited-entity) and [Accredited Entity Type](climate-finance-accredited-entity-type).
     example: |-
```

Comment:
given the mapping is essentially saying refer to these other data points I don't think an example is needed here

Reply:
In this case, I think the example is helpful in order to demonstrate how those other data points combine to provide the necessary information. In particular, the examples for the other data points only show one item in the `finance` array, whereas the example for this data point shows two entries.